dataprofiler/profilers/utils.py

```diff
@@ -483,3 +483,23 @@ def find_diff_of_dicts(dict1, dict2):
         diff = "unchanged"
 
     return diff
+
+def get_capacity(data, unit='M'):
+    """
+    Get size (capacity) of the input data
+
+    :param data: list or array of data
+    :type data: Union[list, numpy.array, pandas.DataFrame]
+    :param unit: capacity unit (B, K, M, or G)
+    :type unit: string
+    :return: capacity of the input data
+    """
+    unit_map = {'B': 0, 'K': 1, 'M': 2, 'G': 3}
+    if unit not in unit_map:
+        raise ValueError('Currently only supports the '
+                         'capacity unit in [B, K, M, G]')
```

Comment:
works, but in future if we wanted this to be more abstract, easier to reference `unit_map` directly so that this ValueError adapts to any changes of `unit_map`.

Not a blocker.

Reply:
good point. Changed to make it works that way